CalPatRec/inc/PhiClusterFinder_types.hh

```diff
@@ -0,0 +1,51 @@
+#ifndef __TrkPatRec_PhiClusterFinder_types_hh__
+#define __TrkPatRec_PhiClusterFinder_types_hh__
+
+#include <vector>
+#include "fhiclcpp/types/Atom.h"
+#include "fhiclcpp/types/Sequence.h"
+
+namespace art {
+  class Event;
+};
+
+namespace fhicl {
+  class ParameterSet;
+};
+
+class TH1F;
+class TH2F;
+
+#include "Offline/RecoDataProducts/inc/StrawHitFlag.hh"
+#include "Offline/RecoDataProducts/inc/StrawHit.hh"
+#include "Offline/RecoDataProducts/inc/StrawHitPosition.hh"
+#include "Offline/RecoDataProducts/inc/TimeCluster.hh"
+#include "Offline/RecoDataProducts/inc/CaloCluster.hh"
+#include "Offline/Mu2eUtilities/inc/LsqSums4.hh"
+
+
+
+namespace mu2e {
+
+  class KalFitResultNew;
```

Comment:
this is not needed here.

Reply:
Yes, removed it. [CalPatRec/inc/PhiClusterFinder_types.hh](https://github.com/Mu2e/Offline/pull/931/files/60d80792e35b9b409fb2c89b187e928a9f543ed8#diff-db4f7728590a32e1a74031a52ad38b4e056b3a34f18f08990cca4865e46d5347)